packages/tokens/build/ios/StyleDictionaryColor.swift

```diff
@@ -0,0 +1,351 @@
+
+//
+// StyleDictionaryColor.swift
+//
+
+//
+
+import UIKit
+
+public enum StyleDictionaryColor {
+    public static let ColorButtonBorderedActiveBackground = [UIColor colorWithRed:0.796f green:0.890f blue:0.710f alpha:1.000f]
```

Comment:
```suggestion
    public static let ColorButtonBorderedActiveBackground = UIColor(red:0.796, green:0.890, blue:0.710, alpha: 1.000)
```

Incorrect Swift syntax and API usage